File: references/MWORKS/MWORKS高校星火计划资料包/培训课程配套材料/01-官网课程配套材料/01-数学与基础应用/03-Julia语法-入门/附件示例/对应课程PPT页码/07_与其他语言的差异/07-2Julia与Python语言的显著差异/page_283/page_283.jl
# 最后一个形参名 args 后加 ... 表示不定长参数 
function add(x, args...)
  res = 0
  for i in args
    res += i
  end
  return res + 2 * x
end
println(add(1, 2, 3, 4, 5)) # 16

File: References/MWORKS/MWORKS高校星火计划资料包/培训课程配套材料/01-官网课程配套材料/01-数学与基础应用/03-Julia语法-入门/附件示例/对应课程PPT页码/07_与其他语言的差异/07-2Julia与Python语言的显著差异/page_283/page_283.py
# 最后一个形参名 args 前加 * 号表示不定长参数
def add(x, *args):
  res = 0
  for i in args:
    res += i
  return res + 2*x
print(add(1, 2, 3, 4, 5)) # 16




